examples/aspnet/Program.cs

```diff
@@ -43,18 +41,6 @@
     await next();
 });
 
-app.MapGet("/openapi.json", async context =>
```

Comment:
We genereren nu deze dynamisch op basis van de endpoints met `app.MapOpenApi("/openapi.json");`